core/src/main/scala/kafka/zk/migration/ZkTopicMigrationClient.scala

```diff
@@ -102,7 +111,7 @@ class ZkTopicMigrationClient(zkClient: KafkaZkClient) extends TopicMigrationClie
 
     val assignments = partitions.asScala.map { case (partitionId, partition) =>
       new TopicPartition(topicName, partitionId) ->
-        ReplicaAssignment(partition.replicas, partition.addingReplicas, partition.removingReplicas)
+        ReplicaAssignment(partition.replicaBrokerIds(), partition.addingReplicas, partition.removingReplicas)
```

Comment:
scala nit: It is usually recommended to not use empty parentless in scala when a parameterless function has no side effect in both the definition of the function and when they call it (which is the case for `partition.replicaBrokerIds`).